### DISCUSSION TITLE
How to Access MainWindow from Another Page?

### DISCUSSION THREAD

**Discussion by ghost1372** [microsoft/WindowsAppSDK#1118] [Q&A] [answered]
I need to call a method (that exist in MainWindow) from another page , in wpf or uwp i can do this by
```
        public static MainWindow Instance;

public MainWindow()
        {
            this.InitializeComponent();
            Instance= this;
        }

```
and:

`MainWindow.Instance.SomeMethod();
`

but this method not working in winui 3.0

**Comment by jaigak** [comment, marked as answer]
You can get the current window and cast it to MainWindow.

**Reply by ghost1372** [reply]
can you write an example?

**Reply by jaigak** [reply]
This should work: 
```
var mainWindow = Window.Current as MainWindow;
// Do anything with it.
```

**Reply by ghost1372** [reply]
oh thank you
It was my fault, my method was private😅
